packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -193,6 +195,40 @@ function DataViews< Item >( {
 		}
 	}, [ hasPrimaryOrLockedFilters, isShowingFilter ] );
 
+	const isInfiniteScroll = useMemo( () => {
+		if ( ! infiniteScrollHandler ) {
+			return false;
+		}
+		return !! view.layout?.infiniteScroll;
+	}, [ view, infiniteScrollHandler ] );
+
+	// Attach scroll event listener for infinite scroll
+	useEffect( () => {
```

Comment:
`useRefEffect` could be an option here instead of `useEffect`, but tbh, it's a very minor difference. Only advantage is you wouldn't need to check `containerRef.current` is set